terraform/core/main.tf

```diff
@@ -200,6 +200,32 @@ resource "google_app_engine_standard_app_version" "fmltc-app-v1" {
     }
   }
 
+  handlers {
+    auth_fail_action = "AUTH_FAIL_ACTION_REDIRECT"
+    login            = "LOGIN_OPTIONAL"
+    security_level   = "SECURE_OPTIONAL"
+    url_regex = "/(.*\\.ico)"
+    static_files {
+      path = "static/\\1"
+      mime_type = "image/x-icon"
+      upload_path_regex = "static/(.*\\.ico)"
+      expiration = "0s"
+    }
+  }
```

Comment:
Did you duplicate the .ico block on purpose?

Reply:
No, I'm not sure how that happened. I fixed it.